internal/expr/parser.go

```diff
@@ -116,18 +116,53 @@ func (sa sliceAccessor) getTypeName() string {
 	return sa.typeName
 }
 
-// columnAccessor stores a SQL column name and optionally its table name.
-type columnAccessor struct {
-	tableName, columnName string
+type columnAccessor interface {
+	String() string
+	tableName() string
+	columnName() string
+}
+
+// staticColumn stores a SQL column name and optionally its table name.
+type staticColumn struct {
```

Comment:
Not sure about the name `staticColumn`, what is static about it? Imo something like `simpleColumn` or `basicColumn` would be better.

Reply:
I tried to convey static as in a regular column name, I agree with you, basicColumn is better.